packages/mui-material/src/Backdrop/Backdrop.d.ts

```diff
@@ -5,9 +5,33 @@ import { TransitionProps } from '../transitions/transition';
 import { Theme } from '../styles';
 import { BackdropClasses } from './backdropClasses';
 import { OverridableComponent, OverrideProps } from '../OverridableComponent';
+import { CreateSlotsAndSlotProps, SlotProps } from '../utils/types';
 
+export interface BackdropSlots {
```

Comment:
We need to add the existing `root` slot to this interface. Copying it from [below](https://github.com/mui/material-ui/blob/aa3bc6e2b4b097e8c1e50d50386d5c3028d5219c/packages/mui-material/src/Backdrop/Backdrop.d.ts#L95) and then removing it from there should be enough.